src/components/DailyTasks.js

```diff
@@ -26,7 +15,7 @@ export default function DailyTasks() {
         </p>
       </header>
       <Wrapper>
-        <TaskList tasks={dailyTasks} />
+        <TaskList todos={todos} setTodos={setTodos} />
```

Comment:
You should not pass down the set function, instead you should create a function that is passed down and call the set function in it.